widget/select_test.go

```diff
@@ -83,6 +83,7 @@ func TestSelect_Tapped_Constrained(t *testing.T) {
 	canvas := fyne.CurrentApp().Driver().CanvasForObject(combo)
 	canvas.(test.WindowlessCanvas).Resize(fyne.NewSize(100, 100))
 
+	combo.Resize(combo.MinSize())
```

Comment:
The addition of this line is problematic, it seems to reflect a change in behaviour.
In fyne_demo go to Graphics, then tap the icon name combo...
You will see that it's sizing up to full size instead of staying the correct width.

Reply:
This line is not the cause of the drop.down size bug. I reported it as #706.
This line is here because the test set-up is uncommon. The combo is not set as canvas' content and therefore never laid out and therefore does have size 0x0.

Reply:
Oh yes, thanks. in that case is the canvas resize really helping? If the component is not inside the canvas then the parent size may have no impact?

Reply:
I don't resize the canvas but the combo :).

Reply:
I was meaning the resize call above, which is a canvas resize (I think it was perhaps put in accidentally?)